core/src/components/loading/usage/react.md

```diff
@@ -7,17 +7,18 @@ import { IonButton, IonContent, IonPage, useIonLoading } from '@ionic/react';
 interface LoadingProps {}
 
 const LoadingExample: React.FC<LoadingProps> = () => {
-  const [present] = useIonLoading();
+  const [present, dismiss] = useIonLoading();
   return (
     <IonPage>
       <IonContent>
         <IonButton
           expand="block"
-          onClick={() =>
+          onClick={() => {
             present({
-              duration: 3000,
```

Comment:
I would rather we keep the original `duration: 3000` property. I don't want users thinking the recommended way to dismiss `IonLoading` is to use a `setTimeout` manually when there's a property built in that will do that.

Reply:
I think GitHub may have accidentally marked this as resolve, but I would still like to see this issue resolved.

One idea is that maybe we could add an additional example underneath the one that uses the `duration` property to show how you can manually dismiss the loading component? You could use your `setTimeout` example there and add a comment above the `setTimeout` that says something like: 

```The `dismiss` method can also be used to dismiss `ion-loading` after a long running task such as a network request.```

Reply:
we could add a comment to mention it
`setTimeout(dismiss, 2000) // simulate a dynamic dismissal of the IonLoading`